workflow/rocoto/gfs_tasks.py

```diff
@@ -894,15 +893,47 @@ def gempak(self):
 
         return task
 
-    def npoess(self):
```

Comment:
No `npoess_pgrb2_0p5deg` job?
https://github.com/NOAA-EMC/global-workflow/blob/dev/gfs.v16/ecf/defs/gfs_v16_3.def#L2190
https://github.com/NOAA-EMC/global-workflow/blob/develop/ecf/scripts/gfs/atmos/gempak/jgfs_atmos_npoess_pgrb2_0p5deg.ecf

Reply:
@KateFriedman-NOAA Are able to confirm that a relative change was (mistakenly) committed?

If not, perhaps we open a separate issue which I can address immediately.

Reply:
Yeah, so looking at the current `gfs_tasks.py` in `develop` I see the `npoess` job:
https://github.com/NOAA-EMC/global-workflow/blob/develop/workflow/rocoto/gfs_tasks.py#L897:L907

Can you add this job back in, update its name to `npoess_pgrb2_0p5deg`, and add the other pieces needed to add it to the rocoto mesh? I added another comment about this job being a dependency for one of the others. Based on ops triggers: https://github.com/NOAA-EMC/global-workflow/blob/dev/gfs.v16/ecf/defs/gfs_v16_3.def#L2193

Thanks!